ocsmesh/cmd.py

```diff
@@ -439,62 +464,60 @@ def _apply_raster_opts(raster, raster_opts):
                 raster.resample(opt)
             else:
                 raster.resample(**opt)
-        if key == 'fill_nodata':
+        if key == "fill_nodata":
             raster.fill_nodata()
-        if key == 'clip':
+        if key == "clip":
             if isinstance(opt, dict):
                 raster.clip(
                     MultiPolygon(
-                        [box(
-                            opt['xmin'],
-                            opt['ymin'],
-                            opt['xmax'],
-                            opt['ymax'])]))
+                        [box(opt["xmin"], opt["ymin"], opt["xmax"], opt["ymax"])]
+                    )
+                )
             else:
                 raise NotImplementedError("clip by geometry")
-        if key == 'chunk_size':
+        if key == "chunk_size":
             raster.chunk_size = opt
 
-        if key == 'overlap':
+        if key == "overlap":
             raster.overlap = opt
 
 
 def _apply_hfun_raster_opts(hfun, hfun_raster_opts):
     for key, opts in hfun_raster_opts.items():
-        if key == 'contours':
+        if key == "contours":
             for kwargs in opts:
                 hfun.add_contour(**kwargs)
-        if key == 'features':
+        if key == "features":
             for kwargs in opts:
                 hfun.add_features(**kwargs)
-        if key == 'subtidal_flow_limiter':
+        if key == "subtidal_flow_limiter":
             if not isinstance(opts, (bool, dict)):
                 raise TypeError(
-                    "subtidal_flow_limiter options must be "
-                    "a boolean or dict.")
+                    "subtidal_flow_limiter options must be " "a boolean or dict."
+                )
             if opts is True:
                 hfun.add_subtidal_flow_limiter()
             else:
                 hfun.add_subtidal_flow_limiter(**opts)
 
 
 def _geom_raster_processing_worker(
-        raster_path,
-        raster_opts,
-        zmin,
-        zmax,
-        join_method,
-        driver,
-        chunk_size,
-        overlap,
+    raster_path,
+    raster_opts,
+    zmin,
+    zmax,
+    join_method,
+    driver,
+    chunk_size,
+    overlap,
```

Comment:
I'm not sure how I feel about removing 1 level of indentation here. It looks OK, but I don't know if it causes more confusion or not do to aligning with function body